src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,73 @@
+@import 'foundation/spacing';
+@import 'shared/typography';
+
+.alert {
+    position:relative;
+    margin-bottom:20px;
+    padding:15px 29px 15px 15px;
```

Comment:
I feel instead of '29px' lets keep it '30px', so that all pixels are near rounded.